keyboard_rewrite/src/lib/y2_keyboard/keyboard_layout.rb

```diff
@@ -0,0 +1,26 @@
+require "cheetah"
+
+module Y2Keyboard
+  class KeyboardLayout
+    attr_reader :code
+    attr_reader :description
+
+    LAYOUT_CODE_DESCRIPTIONS = {
+      "es" => "Spanish",
+      "fr" => "French",
+      "us" => "English (US)"
+    }
+
+    def initialize(code, description)
+      @code = code
+      @description = description
+    end
+
+    def self.load
+      raw_layouts = Cheetah.run("localectl", "list-keymaps", stdout: :capture)
```

Comment:
What about using [memoization](https://en.wikipedia.org/wiki/Memoization) here? You can just cache the value and do not call to `localectl` again.

```ruby
def all
  return @all if @all
  # do things...
  @all = code_with_description.map { ... }
end
```

After all, we do no expect the list of locales to change so often.

Reply:
I think that in this context it is not necessary, because it would be used only once in the module lifetime, and on other hand that will add more complexity to the tests.